src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/index.js

```diff
@@ -11,7 +11,7 @@ export default Form;
 
 export {
     toolbarActionRegistry,
```

Comment:
should we rename the `ToolbarActionRegistry` classes too?

Reply:
If we are already at it, I would say so... Is the situation the same as in the `ToolbarAction` case? I.e. wouldn't it break existing code, if we do that?

Reply:
well it will not break existing code for everyone that imported the registry from the `index.js` file of the admin bundle. it will break existing projects that import the `toolbarActionRegistry` directly. will adjust the `UPGRADE.md` just to be safe.